_scripts/l10n/clone.sh

```diff
@@ -25,7 +25,11 @@ cd "$TARGET_FOLDER"
 # Clone if needed
 clone_opts=(--depth 1)
 do_clone=true
-if [[ -d ".git" && $(git remote -v | grep origin) =~ "$FXA_L10N_REPO" ]]; then
+if [[ -f ".git/index.lock" ]]; then
+    echo "$PREFIX: l10n repo is in locked state! Another git operation is likely progress."
```

Comment:
Just in case someones l10n git folder is or becomes corrupted. Let's provide a slightly more helpful message.